test/Riok.Mapperly.Tests/Mapping/EnumTest.cs

```diff
@@ -213,7 +213,10 @@ public void NullableEnumToOtherNullableEnumShouldCast()
     {
         var source = TestSourceBuilder.Mapping("E1?", "E2?", "enum E1 {A, B, C}", "enum E2 {A, B, C}");
 
-        TestHelper.GenerateMapper(source).Should().HaveSingleMethodBody("return source == null ? default : (global::E2)source.Value;");
+        TestHelper
+            .GenerateMapper(source)
+            .Should()
+            .HaveSingleMethodBody("return source == null ? default(global::E2? ) : (global::E2)source.Value;");

```

Comment:
Do you have an idea why this whitespace in `default(global::E2? )` is added?

Reply:
I was annoyed by this myself, but it turns out the `NormalizeWhitespace()` call in [`GeneratedMethod.ExtractBody()`](https://github.com/riok/mapperly/blob/e073f74390d6ad37b0f1ff8d3d00bf7bbf7f0c72/test/Riok.Mapperly.Tests/GeneratedMethod.cs#L32), is the one adding the extra space:

```csharp
    private static string ExtractBody(MethodDeclarationSyntax declarationSyntax)
    {
        var body =
            declarationSyntax.Body
                ?.NormalizeWhitespace()
                .ToFullString()
                .Trim(' ', '\r', '\n')
                .Trim('{', '}')
                .Trim(' ', '\r', '\n')
                .ReplaceLineEndings() ?? string.Empty;
        var lines = body.Split(Environment.NewLine);
        return lines.Length == 0
            ? string.Empty
            : string.Join(Environment.NewLine, lines.Select(l => l.StartsWith(MethodIndentation) ? l[MethodIndentation.Length..] : l));
    }
```

Not sure if there is a way to control the formatting, or if it matters - as it seems this only affects the tests.

Reply:
Thank you for investigating this. We should probably have a look (in another PR as it is unrelated) if we can remove this `NormalizeWhitespace()` here completely.